From 23c61bda99caa89a0a5132dd3514b9c749e8e9f8 Mon Sep 17 00:00:00 2001
From: worldofpeace <worldofpeace@protonmail.ch>
Date: Wed, 15 May 2019 19:38:00 -0400
Subject: [PATCH] CMake: bump version to 0.5.4

Without this change the application would print an incorrect version in the output.

```
Application.vala:155: NaSC version: 0.4
```

Fixes #114
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eecfb32..90ead5b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,7 +19,7 @@ endif ()
 set (PKGDATADIR "${DATADIR}")
 set (GETTEXT_PACKAGE "nasc")
 set (RELEASE_NAME "nasc")
-set (VERSION "0.4")
+set (VERSION "0.5.4")
 set (VERSION_INFO "Release")
 set (PREFIX ${CMAKE_INSTALL_PREFIX})
 set (DOLLAR "$")
